spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/MissingActuatorTest.java

```diff
@@ -37,13 +37,14 @@
 @ClassPathExclusions({ "spring-boot-actuator-autoconfigure-*.jar", "spring-boot-starter-actuator-*.jar" })
 public class MissingActuatorTest {
 
-	private static ConfigurableApplicationContext getApplicationContext(Class<?> configuration, String... properties) {
```

Comment:
this method `getApplicationContext` was receiving only `Config.class` as parameter of `Class<?> configuration`. So I removed it and inlined it, since I was here anyway.